GOVERNANCE.md

```diff
@@ -0,0 +1,159 @@
+
+A project may have more multiple maintainers, as long as the responsibilities are clearly agreed upon between them. This includes coordinating who handles which issues and pull requests.
+
+Any maintainership change is performed by changing the project's MAINTAINERS.md on the main branch of the repository, and should be announced on the [developers mailing list][devs].
+
+### Technical decisions
+
+Technical decisions that only affect a single project are made informally by the maintainer of this project, and [lazy consensus](#consensus) is assumed.
+
+Technical decisions that span multiple parts of the Prometheus Project are discussed and made on the [Prometheus developer mailing list][devs]. Decisions are usually made by [consensus](#consensus). If no consensus can be reached, the matter may be resolved by [majority vote](#majority-vote).
+
+### New projects
+
+New projects under the Prometheus Project must be proposed to the [developer mailing list][devs] and are subject to [lazy consensus](#consensus).
+
+### Governance changes
+
+Changes to this document are discussed publicly on the [developer mailing mailing list][devs]. Any change requires a [supermajority](#supermajority-vote) in favour.
+
+### Other matters
+
+Any matter that needs a decision, including but not limited to financial matters, may be called to a vote by any member if they deem it necessary. For private or personal matters discussion and voting takes place on the [team mailing list][team], otherwise on the [developer mailing list][devs].
+
+## Voting
+
+The Prometheus Project usually runs by informal consensus, however sometimes a formal decision must be made.
+
+Depending on the subject matter, as laid out in [Decisionmaking](#decisionmaking), different methods of voting are used.
+
+For all votes, voting must be open for at least one week. A vote may be called and closed early if enough votes have come in one way that further votes cannot change the final decision.
+
+In all cases, all and only [team members](#team-members) are eligible to vote, with the sole exception of the forced removal of a team member, in which said member is not eligible to vote.
+
+Discussion and votes on personal matters (including but not limited to team membership and maintainership) are held in private on the [team mailing list][team]. All other discussion and votes are held in public on the [developer mailing list][team].
+
+For public discussions, anyone interested is encouraged to participate. Formal power to object or vote is limited to [team members](#team-members).
+
+### Consensus
+
+The default decision making mechanism for the Prometheus Project is [lazy consensus][lazy]. This means that any decision on technical issues is considered supported by the [team][team] as long as nobody objects.
```

Comment:
is [this link](https://rave.apache.org/docs/governance/lazyConsensus.html) useful, or does it add ambiguity?

Reply:
I like this one a bit more, and it's not part of a retired project: https://couchdb.apache.org/bylaws.html#lazy

Reply:
changed!